assets/js/modules/analytics/components/dashboard/DashboardBounceRateWidget.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * DashboardAllTrafficWidget component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import whenActive from '../../../../util/when-active';
+import PreviewBlock from '../../../../components/preview-block';
+import DataBlock from '../../../../components/data-block';
+import Sparkline from '../../../../components/sparkline';
+import AnalyticsInactiveCTA from '../../../../components/analytics-inactive-cta';
+import { getSiteKitAdminURL, changeToPercent } from '../../../../util';
+import getDataErrorComponent from '../../../../components/notifications/data-error';
+import getNoDataComponent from '../../../../components/notifications/nodata';
+import parseDimensionStringToDate from '../../util/parseDimensionStringToDate';
+
+const { useSelect } = Data;
+
+function DashboardBounceRateWidget() {
+	const {
+		data,
+		error,
+		loading,
+	} = useSelect( ( select ) => {
+		const store = select( STORE_NAME );
+		const args = {
+			dateRange: select( CORE_USER ).getDateRange(),
+			multiDateRange: 1,
+			dimensions: 'ga:date',
+			metrics: [
+				{
+					expression: 'ga:bounceRate',
+					alias: 'Bounce Rate',
+				},
+			],
+		};
+
+		const url = select( CORE_SITE ).getCurrentEntityURL();
+		if ( url ) {
+			args.url = url;
+		}
+
+		return {
+			args,
+			data: store.getReport( args ),
+			error: store.getErrorForSelector( 'getReport', [ args ] ),
+			loading: store.isResolving( 'getReport', [ args ] ),
+		};
+	} );
+
+	if ( loading ) {
+		return <PreviewBlock width="100%" height="202px" />;
+	}
+
+	if ( error ) {
+		return getDataErrorComponent( __( 'Analytics', 'google-site-kit' ), error.message );
```

Comment:
This will need to be updated after the recent changes in #1824, to pass the module slug as first parameter instead. Also it's missing to pass the `error` as last parameter (the signature of this legacy function is super terrible, but let's not worry about this now).